.github/workflows/CI-test.yaml

```
@@ -60,3 +53,45 @@ jobs:
         run: |
           export PREFECT_SERVER_EPHEMERAL_STARTUP_TIMEOUT_SECONDS=300
           PYTHONPATH=src pytest -v --doctest-modules src/
+  format:
```

Suggestion, otherwise I will at some point think that this step is supposed to actually re-format stuff for you:
```suggestion
  check_format:
```